### PR TITLE
Antispoof check empty burst don't propagate

### PR DESCRIPTION
**Pull request by outscale-nts**

### CODE REVIEW

**Comment by outscale-mgo** [comment]
please rebase this on master on on your own branch.

File: src/fail.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+#ifndef _PG_FAIL_H
+#define _PG_FAIL_H
+
+#include <packetgraph/errors.h>
+
+struct pg_brick *pg_fail_new(const char *name, struct pg_error **errp);
+
+#endif /* _FAIL_H */
```
Comment (outscale-jju) [inline src/fail.h]: Bad comment

Comment (outscale-jju) [inline src/fail.h]: Wrong comment

File: tests/antispoof/tests.c [hunk 2]
```diff
@@ -389,7 +425,8 @@ int main(int argc, char **argv)
 			test_antispoof_arp_gratuitous);
 	g_test_add_func("/antispoof/arp/disable",
 			test_pg_antispoof_arp_disable);
-
+	g_test_add_func("/antispoof/mac/burst_empty",
```
Comment (outscale-jju) [inline tests/antispoof/tests.c]: empty_burst ?

**Pull request merged**